core-bundle/tests/Command/UserPasswordCommandTest.php

```diff
@@ -179,6 +179,7 @@ public function testUpdatesTheDatabaseOnSuccess(string $username, string $passwo
                     'password' => '$argon2id$v=19$m=65536,t=6,p=1$T+WK0xPOk21CQ2dX9AFplw$2uCrfvt7Tby81Dhc8Y7wHQQGP1HnPC3nDEb4FtXsfrQ',
                     'locked' => 0,
                     'loginAttempts' => 0,
+                    'pwChange' => false,
```

Comment:
See https://github.com/contao/contao/pull/1754/files#r447678657

Reply:
Done. I also added another test for the password change option.